### PR TITLE
fix: Episode page improved

### PR DESCRIPTION
**Pull request by meowsocute**
Changes made to `/anime/[anime_id]/episodes/[epi_id]`
-> Made episode selection reactive.
-> Also made next/previous buttons interactive.
-> Added new indeterminate animation on video player.

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en" style="background-color: black !important">
+<html lang="en" class="bg-secondary">
```
Comment (baseplate-admin) [inline src/app.html:2]: Please revert this, i had a long long discussion with @moonlitgrace over this

File: src/routes/anime/+layout.svelte [hunk 2]
```diff
@@ -239,8 +239,8 @@
 			</div>
 		</header>
 		<div class="flex h-full w-full flex-auto">
-			<aside class="w-auto flex-none overflow-y-auto overflow-x-hidden">
-				<div class="hidden h-full w-[6.25vw] flex-col justify-between py-[2vw] md:flex">
+			<aside class="fixed flex-none overflow-y-auto overflow-x-hidden w-20">
```
Comment (baseplate-admin) [inline src/routes/anime/+layout.svelte:242]: Uh, Why did we remove the vh?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]
I dont think this approach is right here

File: src/app.html (hunk 1 above)
Comment (moonlitgrace) [reply src/app.html:2]: @meowsocute he is right, this will cause [FOUC](https://en.wikipedia.org/wiki/Flash_of_unstyled_content)

**Pull request closed without merging**